### PR TITLE
Force elasticsearch-py to send queries with body as POST

### PR DESCRIPTION
**Pull request by diox**
Fixes https://bugzilla.mozilla.org/show_bug.cgi?id=1758026

### CODE REVIEW

File: src/olympia/amo/search.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 DEFAULT_TIMEOUT = 5
 
 
-def get_es(hosts=None, timeout=None, **settings):
```
Comment (diox) [inline src/olympia/amo/search.py:17]: `**settings` was never passed anywhere that I could see

Comment (eviljeff) [reply src/olympia/amo/search.py:17]: we don't ever appear to pass `hosts` either, fwiw - and the one place we pass `timeout` as an arg it's `settings.ES_TIMEOUT` which is the default anyway

Comment (eviljeff) [reply src/olympia/amo/search.py:17]: And now I'm looking, the `getattr(...` fallbacks with the constants in this file are also redundant - we define `ES_HOSTS` and `ES_TIMEOUT` in settings_base so those settings are also present.

**Review by eviljeff — APPROVED** [review]
r+wc nits about redundant kwargs and fallbacks

**Pull request merged**